Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/structure.xhtml

```diff
@@ -17,19 +17,22 @@
         xmlns:ui="http://xmlns.jcp.org/jsf/facelets"
         xmlns:p="http://primefaces.org/ui">
     <p:panel id="structureWrapperPanel" styleClass="wrapperPanel">
-        <p:tree value="#{MetadataProcessor.treeNodes}" var="element"
-                selectionMode="single" id="docStruct"
-                selection="#{MetadataProcessor.selectedTreeNode}"
-                draggable="true" droppable="true" dragdropScope="DocStructTree">
-            <p:ajax event="select" listener="#{MetadataProcessor.onNodeSelect}"
-                    update="structureTreeForm:structureWrapperPanel,structureTreeForm:paginationWrapperPanel,metadataWrapperPanel,commentWrapperPanel,galleryWrapperPanel,dialogEditPages"/>
-            <p:ajax event="dragdrop" listener="#{MetadataProcessor.onNodeDragDrop}"
-                    update="structureTreeForm:structureWrapperPanel,structureTreeForm:paginationWrapperPanel,metadataWrapperPanel,commentWrapperPanel,galleryWrapperPanel,dialogEditPages"/>
-            <p:treeNode expandedIcon="ui-icon-document" collapsedIcon="ui-icon-document">
-                <ui:param name="elementType" value="#{element.type.getNameByLanguage(LanguageForm.locale)}"/>
-                <h:outputText value="#{elementType}"/> <h:outputText
-                    value=" #{MetadataProcessor.getLogicalPageNumber(element)}" rendered="#{elementType eq msgs.page}"/>
-            </p:treeNode>
-        </p:tree>
+        <p:repeat var="tree" value="#{MetadataProcessor.trees}">
+            <p:tree value="#{tree}" var="element"
+                    selectionMode="single" id="docStruct"
+                    selection="#{MetadataProcessor.selectedTreeNode}"
+                    draggable="true" droppable="true" dragdropScope="DocStructTree">
+                <p:ajax event="select" listener="#{MetadataProcessor.onNodeSelect}"
+                        update="structureTreeForm:structureWrapperPanel,structureTreeForm:paginationWrapperPanel,metadataWrapperPanel,commentWrapperPanel,galleryWrapperPanel,dialogEditPages"/>
+                <p:ajax event="dragdrop" listener="#{MetadataProcessor.onNodeDragDrop}"
+                        update="structureTreeForm:structureWrapperPanel,structureTreeForm:paginationWrapperPanel,metadataWrapperPanel,commentWrapperPanel,galleryWrapperPanel,dialogEditPages"/>
+                <p:treeNode expandedIcon="ui-icon-document" collapsedIcon="ui-icon-document">
+                    <ui:param name="elementType" value="#{element.type.getNameByLanguage(LanguageForm.locale)}"/>
+                    <h:outputText value="#{elementType}"/> <h:outputText
+                        value=" #{MetadataProcessor.getLogicalPageNumber(element)}" rendered="#{elementType eq msgs.page}"/>
+                    <h:outputText value=" 🔗" rendered="#{MetadataProcessor.isLink(element)}"/>
```

Comment:
this seems to be a strange value

Reply:
You should use fontawesome... Just take random hyperlink icon. But never use such a thing like here!